src/services/cocktails.service.ts

```diff
@@ -3,18 +3,17 @@ import { HttpClient  } from '@angular/common/http';
 import { Injectable } from '@angular/core';
 import { pluck } from 'rxjs/operators';
 import { Observable } from 'rxjs/Observable';
-import { CocktailsList, CocktailDetails, IngredientItem } from '../app/app.models';
+import { CocktailsList, CocktailDetails, IngredientItem, Categories } from '../app/app.models';
 import { DATA_URL } from '../config/api';
 
 const DATA_KEY = 'drinks';
-
 @Injectable()
 export class CocktailsService {
   constructor (private http: HttpClient) {}
 
-  getCocktails(): Observable<CocktailsList> {
-    const response = this.http.get(DATA_URL.COCKTAILS);
-    return response.pipe(pluck(DATA_KEY));
+  getCocktails(category): Observable<CocktailsList> {
+    const response = this.http.get(DATA_URL.COCKTAILS + category);

```

Comment:
will be great to replace this by string interpolation ```const response = this.http.get(`DATA_URL.COCKTAILS category`);```

Reply:
done